### PR TITLE
Removed oriel implementation

### PR DESCRIPTION
**Pull request by jeteve**
Removing Oriel code.

See https://trello.com/c/mapMRS0N

### CODE REVIEW

File: common/app/common/configuration.scala [hunk 1]
```diff
@@ -244,11 +244,6 @@ class GuardianConfiguration extends Logging {
     lazy val subscribeWithGoogleApiUrl = configuration.getStringProperty("google.subscribeWithGoogleApiUrl").getOrElse("https://swg.theguardian.com")
   }
 
-  object oriel {
-    lazy val orielApiKey = configuration.getStringProperty("oriel.api.key")
```
Comment (kelvin-chappell) [inline common/app/common/configuration.scala]: Will need to remove this from the parameter store.

**Review by kelvin-chappell — COMMENTED** [review]
It looks good to me, but @janua is probably best to give thumbs up

File: common/app/common/configuration.scala (hunk 1 above)
Comment (jeteve) [reply common/app/common/configuration.scala]: Can you point me to it?

Comment (kelvin-chappell) [reply common/app/common/configuration.scala]: I can try ...

**Comment by janua** [comment]
:+1:

**Pull request merged**